docs/0.1/creating_products.md

```diff
@@ -0,0 +1,313 @@
+   circuit can view the product data.
+
+   **Note**: This command does not display any output. Instead, check the log
+   messages on the console (or the terminal window where you started the Grid
+   Docker environment) for the success or failure of this operation.
+
+### Display Product Information
+
+{:start="7"}
+
+7. List all existing products to verify that the new product has been added.
+
+   ```
+   root@gridd-alpha:/# grid product list
+   Product Id: "723382885088"
+    Product Type: "GS1"
+    Owner: "314156"
+    Properties:
+           Property Name: "species"
+           Data Type: "String"
+           Bytes Value: Some([])
+           Boolean Value: Some(false)
+           Number Value: Some(0)
+           String Value: Some("tuna")
+           Enum Value: Some(0)
+           Struct Values: Some([])
+           Lat/Lon Values: Some(LatLong { latitude: 0, longitude: 0 })
+
+           Property Name: "length"
+           Data Type: "Number"
+           .
+           .
+           .
+   ```
+
+1. (Optional) You can connect to a different node and repeat the last two
+   commands to verify that the product has been shared with all nodes on the
+   circuit.
+
+    a. Open a new terminal and connect to the other node's `gridd` container
+       (such as `gridd-beta`).
+
+      ```
+      $ docker exec -it gridd-beta bash
+      ```
+
+    b. Set the `GRID_SERVICE_ID` environment variable to the full service ID
+       for this node (such as `01234-ABCDE::xyBB`).
+
+      ```
+      root@gridd-beta:/# export GRID_SERVICE_ID="01234-ABCDE::xyBB"
+      ```
+
+    c. Display all products. The output should be the same as on the first node.
+
+      ```
+      root@gridd-beta:/# grid product list
+      Product Id: "723382885088"
+       Product Type: "GS1"
+       Owner: "314156"
+       Properties:
+              Property Name: "species"
+              Data Type: "String"
+              Bytes Value: Some([])
+              Boolean Value: Some(false)
+              Number Value: Some(0)
+              String Value: Some("tuna")
+              Enum Value: Some(0)
+              Struct Values: Some([])
+              Lat/Lon Values: Some(LatLong { latitude: 0, longitude: 0 })
+
+              Property Name: "length"
+              Data Type: "Number"
+              .
+              .
+              .
+      ```
+
+### Update a Product
+
+To update a product, you must be an agent for the product owner (the
+organization that is identified in the product definition).
+
+1. Modify the definition in the product YAML file (such as `product.yaml`).
+
+   You don't have to use the same file that was used to create the product,
+   but the file must specify the same information (product ID, type, and owner)
+   and present the product properties in the same order.
+
+1. Use the `update` subcommand for `grid product` to submit the changes
+   to the distributed ledger.
+
+   ```
+   root@gridd-beta:/# grid product product.yaml
```

Comment:
This should be `grid product update product.yaml`

Reply:
Fixed.